code-challenges.rb

```diff
@@ -21,8 +32,11 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+# def vowel_destroyer (stringArr)
+#     p stringArr.delete "aeiouAEIOU"
+# end
 
-
+# vowel_destroyer(album3)
```

Comment:
Looks great. A good practice is to test your method by having the `p` on the method invocation rather than the inner workings of the method.